server/api/balancer.go

```diff
@@ -37,7 +39,7 @@ func newBalancerHandler(svr *server.Server, rd *render.Render) *balancerHandler
 	}
 }
 
-func (h *balancerHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+func (h *balancerHandler) Get(w http.ResponseWriter, r *http.Request) {
```

Comment:
can we add a comment for how to send these requests? It may be hard for user to communicate with PD later with HTTP. 
@qiuyesuifeng